src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesSequences.js

```diff
@@ -63,7 +63,11 @@ class SeriesSequences extends Component {
 }
 
 SeriesSequences.propTypes = {
-  relations: PropTypes.object.isRequired,
+  relations: PropTypes.object,
```

Comment:
Are you sure about this change? In general, IMHO, we should have required as much as possible, and limit the use of `_get` to what is really optional.
If this component should not render when there are no relations, it should be handled when calling it...

Reply:
When I am accessing a `Serial` I get Warning: Failed prop type: The prop `relations` is marked as required in `SeriesSequences`, but its value is `undefined`. That is why I changed it, I can revert it back and handle all these props issues together in an issue.